numpy/lib/stride_tricks.py

```diff
@@ -1,8 +1,7 @@
 """
 Utilities that manipulate strides to achieve desirable effects.
 
-An explanation of strides can be found in the "ndarray.rst" file in the
-NumPy reference guide.
+An explanation of strides can be found in the :ref:`arrays.ndarray`.
```

Comment:
FWIW I don't think the module docstring is included in the reference docs, so this won't have any effect.